src/main/groovy/org/labkey/gradle/plugin/TeamCity.groovy

```diff
@@ -108,10 +108,6 @@ class TeamCity extends Tomcat
                 task.group = GroupNames.TEST_SERVER
                 task.description = "Removes log files from Tomcat and TeamCity"
                 task.dependsOn project.tasks.cleanLogs
-                if (!BuildUtils.useEmbeddedTomcat(project)) {
-                    // Not valid for embedded Tomcat
-                    task.dependsOn project.tasks.cleanTemp
```

Comment:
Can this task definition (in the `Tomcat` plugin) be removed if not valid for embedded?  That would also cascade to removing at least the `validateCatalinaHome` method from the `TomcatExtension`.

Reply:
Removed, which led to removing what seems like the primary use of `TomcatExtension` (in `Tomcat.groovy`). One reference remains, in `RunUiTest.groovy`. Unclear if we can remove that as well (and the extension).